praetorian_cli/ui/aegis/commands/proxy.py

```
@@ -0,0 +1,337 @@
+
+    try:
+        public_hostname = agent.health_check.cloudflared_status.hostname
+    except Exception:
+        menu.console.print(f"[{colors['error']}]Cannot determine agent hostname[/{colors['error']}]")
+        menu.pause()
+        return
+
+    if not user:
+        try:
+            _, user = menu.sdk.aegis.api.get_current_user()
+        except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)